.tekton/push.yaml

```
@@ -132,7 +132,12 @@ spec:
                 DATA_BUNDLE_TAG=$(date '+%s')
                 export DATA_BUNDLE_REPO DATA_BUNDLE_TAG
 
-                .tekton/scripts/build-acceptable-bundles.sh "$@"
+                list=()
+                for f in "$@"; do
+                  [[ -f "$f" ]] && list+=("$f")
+                done
+
+                .tekton/scripts/build-acceptable-bundles.sh "${list[*]}"
```

```suggestion
                .tekton/scripts/build-acceptable-bundles.sh "${list[@]}"
```
unless there's a reason to expand the array into a string?

---

I always confuse these two, thanks!